internal/horusec-javascript/ast.go

```diff
@@ -332,7 +332,58 @@ func (p *parser) parseStmt(node *cst.Node) ast.Stmt {
 		}
 
 		return stmt
+	case SwitchStatement:
+		// TODO: handle switch cases with labels
+		// inner: switch (i) {
+		// case 1:
+		//	 break inner
+		// }
+		stmt := &ast.SwitchStatement{
+			Position: ast.NewPosition(node),
+			Value:    p.parseExpr(node.ChildByFieldName("value")),
+			Body:     p.parseFuncBody(node.ChildByFieldName("body")),
+		}
 
+		return stmt
+	case SwitchCase:
+		stmt := &ast.SwitchCase{
+			Position: ast.NewPosition(node),
+			Value:    p.parseExpr(node.ChildByFieldName("value")),
+		}
+
+		var body []ast.Stmt
```

Comment:
We can declared this body with a size:
`body := make([]ast.Stmt, 0, node.NamedChildCount()-1) // Minus 1 here because we start the iteration on the second node`